deployment/server.bc.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
me being nit picky but  i think this shouldn't be referred to as a  server.  A lot of microservices  run  under a 'server'. I think the  templates should be named after the type of microservice they offer.